### ISSUE TITLE
Headings are very large for reports

### ISSUE THREAD

**Issue by knutzk** [kzoch/kTheTex-bundle#13]
As the report class currently uses the same settings for headings as the thesis class, they are typeset in quite large font sizes. Does it maybe make sense to remove the chapter level internally and only use sections as the maximal size? The ideal way would be to let the user choose by class option which level should be the top level, e.g. `toplevel=chapter`, `toplevel=section`. And then adjust the numbering schemes accordingly.

For using chapters as the top level, there shouldn't be a `\cleardoublepage` command for every single new chapter. This can be achieved with the KOMA option `open=any` which should be set as a standard value for the report class.

**Comment by knutzk** [comment]
With the following code, chapters can be omitted completely (the user still uses the commands `\chapter`, `\section` etc. though):
```
\renewcommand*\thesection{\arabic{section}}
\let\chapter\section
\let\section\subsection
\let\subsection\subsubsection
\numberwithin{equation}{section}
\numberwithin{table}{section}
\numberwithin{figure}{section}
```
This renews the definitions of the sectioning commands in such a way that the top level can't be accessed anymore. The `renecommand` removes the chapter numbers from the headlines (which would be a leading 0). The last three commands adjust the numbering for the floats.

**Comment by knutzk** [comment]
This is being addressed in pull request #19.
